staging/src/kubevirt.io/client-go/kubecli/version_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("Kubevirt Version Client", func() {
 				ghttp.RespondWithJSONEncoded(http.StatusOK, groupInfo),
 			),
 			ghttp.CombineHandlers(
-				ghttp.VerifyRequest("GET", "/apis/"+groupInfo.PreferredVersion.GroupVersion+"/version"),
+				ghttp.VerifyRequest("GET", "/apis"+groupInfo.PreferredVersion.GroupVersion+"/version"),
```

Comment:
How come we don't need this "/"?

Reply:
Yeah, looks a bit weird. It appears that we use `/apis//apis/subresources.kubevirt.io/v1alpha3/version` uri (with `//`):

https://github.com/kubevirt/kubevirt/blob/0f2c6714ae95371f14b34361f747c2311df18a4d/staging/src/kubevirt.io/client-go/kubecli/version.go#L79-L83 

Here `group.PreferredVersion.GroupVersion` already contains a leading `/` (potentially we can fix also the format if it makes sense).

Previously, it was passed to the `RequestURI` method, which under the hood calls `url.Parse(uri)`. This one preserves the path as-is with `//`. Hence, we needed this extra `/` in the test.

With this patch, the uri is now passed to `AbsPath`, which does `path.Join(..., uri)`. It looks like this join sanitizes the resulting path and the output has only a single `/`.

Simple reproducer: https://go.dev/play/p/IU80SS_Gj5v